webapps/src/main/webapp/vue-apps/automatic-translation-administration/components/AutomaticTranslationAdministrationApp.vue

```diff
@@ -104,7 +104,7 @@ export default {
       getConfiguration().then(data => {
         this.connectors=data.connectors;
         this.connectors.unshift({'name': 'none','description': this.$t('automatic.translation.administration.noconnectorselected')});
-        this.selectedConnector = data.active === null ? 'none' : data.active;
+        this.selectedConnector = data.activeConnector === null ? 'none' : data.activeConnector;
```

Comment:
```suggestion
        this.selectedConnector = data && data.activeConnector || 'none';
```
It's better to use falsy and truthy expressions for better code flexibility.

Reply:
done